network-protocols.md

```diff
@@ -62,6 +62,34 @@ func SayHello(p PeerID) {
 
 Upon receiving a 'hello' stream from another node, you should read off the CBOR RPC message, and then check that the genesis hash matches you genesis hash. If it does not, that node is not part of your network, and should probably be disconnected from. Next, the `HeaviestTipSet`, claimed `HeaviestTipSetHeight`, and peerID of the other node should be passed to the chain sync subsystem.
 
+# Heartbeat
```

Comment:
Unclear that this should not just be the hello handshake

Reply:
I don't think we need an explicit heartbeat thing

Reply:
Agreed perhaps in form, though it would be helpful to have some functionality to periodically query peers to make sure they aren't tracking a heavier chain than you... Thoughts?

Reply:
silly q. block propagation takes care of this